examples/grpc-js/helloworld_grpc_pb.js

```diff
@@ -15,50 +15,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-
 'use strict';
+var grpc = require('@grpc/grpc-js');
```

Comment:
All of the changes to this file appear to be unrelated to the purpose of the PR?

Reply:
This is the recompiled output from protoc. Would you prefer me to manually edit the file to make it work?

Reply:
I didn't realize this was generated

Reply:
Yea both the grpc and grpc-js repos use generated code from protoc, using a proto file from a grpc example on github.com/grpc/grpc